components/card/index.jsx

```diff
@@ -0,0 +1,47 @@
+import { createElement } from "complate-stream";
+
+export function Card(_, ...children) {
+	return <div class="card">{ children }</div>;
+}
+
+export function CardImg({ src, alt }, ...children) {
+	return <img class="card-img-top" src={ src } alt={ alt }/>;
+}
+
+export function CardHeader({ level = "4" }, ...children) {
+	let attrs = {
+		class: "card-header"
+	};
+
+	return createElement(`h${level}`, attrs, children);
+}
+
+export function CardBody(_, ...children) {
+	return <div class="card-body">{ children }</div>;
+}
+
+export function CardTitle({ level = "5" }, ...children) {
```

Comment:
What do you think about also adding `additionalClasses` here? The Bootstrap examples don’t use it, but it might still make sense!

Reply:
Also if we did this, we’d have the same code as  for `CardSubtitle` except with a different default level, which seems conceptionally sound.

Reply:
I guess almost all elements should allow `additionalClasses`?

Reply:
I believe they should, and possibly more (see #11).